<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="logolabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>-30</y>
     <width>400</width>
     <height>225</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;微软雅黑 Light&quot;;</string>
   </property>
   <property name="text">
    <string>Student Achievement Management System</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>331</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="loginbtn">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>LOGIN</string>
      </property>
      <property name="icon">
       <iconset resource="res.qrc">
        <normaloff>:/res/img/logo.png</normaloff>:/res/img/logo.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="registerbtn">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_stu">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>Student</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_5">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_tea">
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>Teacher</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="useridline">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>100000</string>
   </property>
   <property name="placeholderText">
    <string>Please enter the ID</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordline">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>123</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Please enter password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_head">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
